.dockerignore

```diff
@@ -1,3 +1,12 @@
-bothan-api/server/config.toml
-bothan-api-proxy/config.toml
-target
+debug/
+target/
+
+Cargo.lock
```

Comment:
Cargo.lock should not be ignored if we are not writing a library. https://doc.rust-lang.org/cargo/faq.html#why-have-cargolock-in-version-control

Same for .gitignore (see comment there)

Reply:
why resolved na

Reply:
its changes will be pushed at once after resolving all